mappings/net/minecraft/nbt/AbstractNbtList.mapping

```diff
@@ -13,8 +13,8 @@ CLASS net/minecraft/unmapped/C_zveiijin net/minecraft/nbt/AbstractNbtList
 	METHOD m_wjgcgvzd addElement (ILnet/minecraft/unmapped/C_oivssbvb;)Z
 		ARG 1 index
 		ARG 2 nbt
-	METHOD remove remove (I)Ljava/lang/Object;
+	METHOD remove (I)Ljava/lang/Object;
 		ARG 1 index
-	METHOD set set (ILjava/lang/Object;)Ljava/lang/Object;
+	METHOD set (ILjava/lang/Object;)Ljava/lang/Object;
```

Comment:
```suggestion
	METHOD set set (ILjava/lang/Object;)Ljava/lang/Object;
```
These three seem to be bridges or something:
![image](https://github.com/user-attachments/assets/00bcb154-4c90-4b95-87b6-a133c5396b12)